packages/pds/src/db/migrations/20230509T192324175Z-seq-invalidated.ts

```diff
@@ -0,0 +1,43 @@
+import { Kysely } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('repo_seq')
+    .dropConstraint('invalidated_by_fkey')
+    .execute()
+  await db.schema.alterTable('repo_seq').dropColumn('invalidatedBy').execute()
+  await db.schema
+    .alterTable('repo_seq')
+    .addColumn('invalidated', 'int2', (col) => col.notNull().defaultTo(0))
+    .execute()
+}
+
+export async function down(
+  db: Kysely<unknown>,
+  dialect: Dialect,
+): Promise<void> {
+  await db.schema.alterTable('repo_seq').dropColumn('invalidated').execute()
+
+  if (dialect === 'pg') {
+    await db.schema
+      .alterTable('repo_seq')
+      .addColumn('invalidatedBy', 'bigint')
+      .execute()
+  } else {
+    await db.schema
+      .alterTable('repo_seq')
+      .addColumn('invalidatedBy', 'integer')
+      .execute()
+  }
+  await db.schema
+    .alterTable('repo_seq')
+    .addForeignKeyConstraint(
+      'invalidated_by_fkey',
+      // @ts-ignore
+      ['invalidatedBy'],
+      'repo_seq',
+      ['seq'],
+    )
```

Comment:
I don't think this is supported by sqlite— probably fine to just skip in that case.